assets/js/flattenPremisesOptions.js

```diff
@@ -0,0 +1,109 @@
+function flattenPremisesOptions(select, index) {
+  var formGroup = select.parentNode
```

Comment:
I don't think there's much benefit in changing it but for future ref I think it's safe to use `let`/`const` for variables in clientside JS since we no longer support IE11